### ISSUE TITLE
Need ability to specify different sigma value for z

### ISSUE THREAD

**Issue by jni** [ilastik/ilastik#100]
Many images are 3D but non-isotropic. It would be useful to be able to set the sigma for each dimension independently in these cases.

**Comment by christophdecker** [comment]
It's not a pleasure to dig out this almost 3 years old issue, but it would be really nice to have this in the ilastik GUI, because there are many anisotropic datasets with lower resolution in z-direction.

**Comment by burgerdev** [comment]
Are we talking about the thresholding applet?

**Comment by christophdecker** [comment]
No, it's about the feature computation for pixel classification. E.g. the Gaussian smoothing kernel should have a different sigma in z-direction than in x- and y-direction, if the data is anisotropic in z-direction.

**Comment by burgerdev** [comment]
Ah, I was thinking of something else. Nevermind :)

**Comment by jni** [comment]
@ilastikdev any update on this? I was recently helping someone put their anisotropic 3D fluorescence stack in Ilastik, and things work (and allowing feature computation in 2D helps a lot already, thanks!), but it would be super nice if the xz/yz views weren't squashed, as well as the 3D object view.
